### PR TITLE
Feat/21 contact section of website

### PR DESCRIPTION
**Pull request by jiayuu-u**
### Descriptions

- Link to UI design (if possible):
image

### Provide screenshots or videos that the function is working.
image

### Provide screenshots of no warning, or error in the console.
image

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

**Review by Cameron933 — APPROVED** [review]
LGTM

File: components/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import styles from "@/styles/Button.module.css";
+import { textFont } from "@/pages/_app";
+
+const Button = ({ text }: { text: string }) => {
```
Comment (tatoMa) [inline components/Button.tsx:4]: It is good for now. But consider what if the props are not only about the text but also with icons or even no text props.

**Review by tatoMa — COMMENTED** [review]
Do you think putting the Button and Contact components in the same folder inside of the components folder is a good idea?

File: styles/Button.module.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  width: 102px;
+  height: 35px;
+  background-color: transparent;
+  color: var(--text-default);
+  border-radius: 6px;
+  border: 1px solid var(--border-default);
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  cursor: pointer;
+}
```
Comment (tatoMa) [inline styles/Button.module.css:25]: Any hover or active state of these buttons?

File: styles/Contact.module.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
+.contact {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
```
Comment (tatoMa) [inline styles/Contact.module.css:3]: Nice grid.

**Review by HarryuCATGREY — APPROVED** [review]
LGTM

File: components/Button.tsx (hunk 1 above)
Comment (Cameron933) [reply components/Button.tsx:4]: image
Can we do it this way？

File: components/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import styles from "@/styles/Button.module.css";
+import { textFont } from "@/pages/_app";
+
+const Button = ({ children, ...otherProps }: { children: any }) => {
```
Comment (tatoMa) [inline components/Button.tsx:4]: The type of children can not be any.

File: styles/Button.module.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  cursor: pointer;
+}
+
+.default_button:hover {
+  color: var(--text-default);
+  background-color: transparent;
+  border: 1px solid var(--border-default);
+}
```
Comment (tatoMa) [inline styles/Button.module.css:18]: What about the active state?

**Review by tatoMa — CHANGES_REQUESTED** [review]

**Review by tatoMa — APPROVED** [review]

**Pull request merged**